lib/vanagon/project/dsl.rb

```diff
@@ -171,6 +171,21 @@ def generate_archives(archive)
         @project.compiled_archive = archive
       end
 
+      # Sets the release for the project to the number of commits since the
+      # last tag. Requires that a git tag be present
+      # and reachable from the current commit in that repository.
+      #
+      def release_from_git
+        git_version = Git.open(File.expand_path("..", Vanagon::Driver.configdir)).describe('HEAD', tags: true)
+        # git describe actually contains the information we need here.
+        # for example: 0.15.2-5-g332dbcc, the -5- indicates there are 5
+        # commits since the last tag, so just split on '-' and take the second
+        # output
```

Comment:
[nitpick] you're taking the second from the last output, which is probably the correct behavior (in case the tag has a `-` in it), but the comment doesn't match actual behavior.